README.md

```diff
@@ -66,4 +66,4 @@ hugo serve
 
 The lighthouse auditing is done through https://web.dev. In an effort to make lighthouse auditing a fair assessment of the template rather than server set up, `?nocache=true` is appended to all audited URLs, disabling server side caching.
 
-Note that the performance value does vary slightly with each audit. If you feel an audit is inaccurate, you can submit a PR deleting the file of the the specific template from the data folder. The template demo will be re-audited automatically upon the submission of the PR. The **whole process of auditing** is transparent (the code is [here](https://github.com/ciar4n/themeglobe/blob/master/scripts/capture-lighthouse.js)) and is possible thanks to Github actions.
+Note that the performance value do vary slightly with each audit. If you feel an audit is inaccurate, you can submit a PR deleting the file of the the specific template from the data folder. The template demo will be re-audited automatically upon the submission of the PR. The **whole process of auditing** is transparent (the code is [here](https://github.com/ciar4n/themeglobe/blob/master/scripts/capture-lighthouse.js)) and is possible thanks to Github actions.
```

Comment:
Replace `value` with `values` and remove the word `do`

Reply:
Thanks